src/router/PublicRoute.js

```diff
@@ -8,7 +8,7 @@ import { ACTIVITY_PLAN_ROUTE } from '../constants/routes';
 export const PublicRoute = ({ isAuthenticated, component: Component, ...rest }) => (
   <Route
     {...rest}
-    component={props => {
+    render={props => {
```

Comment:
why not tenary?

Reply:
I used 'if' intentionally. There is similar component in PrivateRoute and there I have this annoying eslint-prettier issue with indentation in ternary operator broken into multiple lines. Last time you said to ignore it and use 'if' instead of fighting with configuration. In this file ternary operator is working correctly but i changed both to 'if' for consistency.